colossalai/tensor/module_utils.py

```diff
@@ -0,0 +1,93 @@
+from typing import Dict
+from colossalai.tensor import ColoParameter, ParallelAction, TensorSpec
+import torch
+
+_COLOSSAL_MODULES: Dict[str, object] = {}
+
+
+def register_colo_module(module: torch.nn.Module, colo_module):
```

Comment:
type hinting is wrong!
the 1st arg should be Type[torch.nn.Module] but not a module instance?

Reply:
Also what is the type of colo_module?

Reply:
Right. I will fix it.

Reply:
Done.